packages/libxml2.rb

```diff
@@ -35,8 +43,13 @@ def self.patch
   end
 
   cmake_options '-DLIBXML2_WITH_PYTHON=OFF \
```

Comment:
Why is python off?

Reply:
That's built in `py3_libxml2.rb` .

No reason that has to happen though...

Reply:
I removed py3_libxml2...